### ISSUE TITLE
Multiple Responses Same Path

### ISSUE THREAD

**Issue by timrijckaert** [andrzejchm/RESTMock#46]
Hiya, 

Awesome library! 
However.

We are in a situation where we want to test if a pull to refresh succeeds successfully.
See pseudo code 

```kotlin
fun onPullToRefreshTheDataShouldBeDifferent() {
        whenGET(pathEndsWith("/some/path")
                .thenReturnFile("/file_with_json_1.json")
                .thenReturnFile("/file_with_json_2.json"

        launchActivity()
        doAssertionViewHolder()

        swipeToRefresh()

       doAssertionDifferentViewHolderShouldNowBeVisible()
}
```

The `RESTMockWebServer` seems to always return the first defined `JSON` ("/file_with_json_1.json").
Does this work like intended?  
Because I don't see any samples like this on your repo

Thanks In advance.

Tim

**Comment by andrzejchm** [comment]
Each consecutive call to `then*()` method is ignored indeed, this is a design flaw that would be fixed with the next version of the library.

The best way to fix it would be to make it work like `then*` methods from Mockito.

**Comment by matoelorriaga** [comment]
Hi! I'm really looking forward for this feature! Thanks

**Comment by andrzejchm** [comment]
implemented in `0.2.0` version, enjoy!
